src/input/label.component.ts

```diff
@@ -169,9 +169,16 @@ export class Label implements AfterContentInit, AfterViewInit {
 	 */
 	ngAfterViewInit() {
 		if (this.wrapper) {
-			const inputElement = this.wrapper.nativeElement.querySelector("input,textarea,div");
```

Comment:
Why are we querying for a div anyways? I'm keeping it for the time being since it's considered a breaking change if I remove it.